packages/java/parser-jvm-plugin-nonnull/src/main/java/com/vaadin/hilla/parser/plugins/nonnull/NonnullPluginConfig.java

```diff
@@ -72,7 +72,9 @@ static class Processor extends ConfigList.Processor<AnnotationMatcher> {
                 new AnnotationMatcher("androidx.annotation.NonNull", false, 30),
                 new AnnotationMatcher("org.eclipse.jdt.annotation.NonNull",
                         false, 30),
-                new AnnotationMatcher("com.vaadin.hilla.Nonnull", false, 30));
+                new AnnotationMatcher("com.vaadin.hilla.Nonnull", false, 30),
+                new AnnotationMatcher("org.jspecify.annotations.NonNull", false,
```

Comment:
Should also add the corresponding `Nullable` from the same package.

Reply:
added